docs/installingNative.md

```diff
@@ -41,18 +46,6 @@ This page describes the procedure to install and run PrairieLearn without any us
   export LDFLAGS="-L$(brew --prefix graphviz)/lib"
   ```
 
-- Create the database (one time only):
-
-  ```sh
-  initdb -D ~/defaultdb
-  ```
-
-- Run the database:
-
-  ```sh
-  pg_ctl -D ~/defaultdb -l ~/logfile start
-  ```
-
```

Comment:
AFAIK I've never had to do this. On macOS, Homebrew will generally run Postgres for you at boot, and on Linux, I'm pretty sure something similar happens too.

Reply:
@nwalters512 A little late on this, but I've had to run these commands every time I restart my machine (Linux). Could these be added back? I'm not sure if this is just a Linux thing, but in addition, I usually have some weird issue with permissions and have to spend time troubleshooting the database start if it's been a while.